hw_diag/utilities/miner.py

```diff
@@ -8,9 +10,12 @@ def fetch_miner_data(diagnostics):
     try:
         peerbook = client.get_peer_book()[0]
         height = client.get_height()
+    except MinerFailedFetchData as err:
+        LOGGER.exception(err)
```

Comment:
With this change, `MinerFailedFetchData` is being swallowed. tasks.py won't know if this error happens and won't be able to update the keys like MC, MD, etc correctly.

Reply:
Thank you for catching this, with the tree of exceptions it gets confusing sometimes 😂